### PR TITLE
zookeeper: minor no-op cleanup

### PR DESCRIPTION
**Pull request by derekargueta**
Description:
1) remove un-necessary imports & bazel dependencies
2) remove un-necessary namespace prefix in `zookeeper_config.cc`
3) minor formatting nit in `zookeeper_filter.cc`
4) add missing stdlib imports (IWYU)
Risk Level: Low
Testing: Existing
Docs Changes: N/A
Release Notes: N/A

cc @rgs1 

Signed-off-by: Derek Argueta <dereka@pinterest.com>

### CODE REVIEW

File: source/extensions/filters/network/zookeeper_proxy/zookeeper_filter.h [hunk 1]
```diff
@@ -63,7 +67,7 @@ struct ZooKeeperProxyStats {
  */
 class ZooKeeperFilterConfig {
 public:
-  ZooKeeperFilterConfig(const std::string &stat_prefix, uint32_t max_packet_bytes, Stats::Scope& scope);
+  ZooKeeperFilterConfig(const std::string& stat_prefix, uint32_t max_packet_bytes, Stats::Scope& scope);
```
Comment (rgs1) [inline source/extensions/filters/network/zookeeper_proxy/zookeeper_filter.h]: huh how come format test misses this? can we fix that test so it catches this?

**Comment by rgs1** [comment]
Looks great! Wonder if we can improve format/clang_tidy to catch all of these....

**Comment by derekargueta** [comment]
coverage failure seems unrelated
```
./test/common/grpc/grpc_client_integration_test_harness.h:243: Failure
Failed
Test timeout
```

File: source/extensions/filters/network/zookeeper_proxy/zookeeper_filter.h (hunk 1 above)
Comment (derekargueta) [reply source/extensions/filters/network/zookeeper_proxy/zookeeper_filter.h]: Yeah I'm also puzzled... it's also messing up import headers. Example in `zookeeper_config.cc` from (main header at top)
```
#include "extensions/filters/network/zookeeper_proxy/zookeeper_config.h"

#include "envoy/config/filter/network/zookeeper_proxy/v1alpha1/zookeeper_proxy.pb.validate.h"
#include "envoy/registry/registry.h"
#include "envoy/server/filter_config.h"

#include "common/common/logger.h"

#include "extensions/filters/network/zookeeper_proxy/zookeeper_filter.h"
```

to (headers always alphabetical)
```
#include "envoy/config/filter/network/zookeeper_proxy/v1alpha1/zookeeper_proxy.pb.validate.h"
#include "envoy/registry/registry.h"
#include "envoy/server/filter_config.h"

#include "common/common/logger.h"

#include "extensions/filters/network/zookeeper_proxy/zookeeper_config.h"
#include "extensions/filters/network/zookeeper_proxy/zookeeper_filter.h"
```

Comment (snowp) [reply source/extensions/filters/network/zookeeper_proxy/zookeeper_filter.h]: I think the trailing \ on https://github.com/envoyproxy/envoy/blob/master/source/extensions/filters/network/zookeeper_proxy/zookeeper_filter.h#L51 is disabling clang-format for this code. Could you try removing it and running clang-format to see if that fixes it?

Comment (derekargueta) [reply source/extensions/filters/network/zookeeper_proxy/zookeeper_filter.h]: 🤦‍♂️ yep that was it! Also fixes a few of other formatting things. Thanks @snowp!

**Comment by derekargueta** [comment]
@rgs1 WDYT about removing the `zookeeper_` prefix from the file names? A bit redundant since it's in the `zookeeper_proxy` directory. MySQL is the only other filter that prefixes like that (which I understand is what you based this filter off of)

**Comment by rgs1** [comment]
Yeah I like that idea

**Comment by snowp** [comment]
Mind merging master to pick up the fix for coverage?

**Pull request merged**